applications/TopologyOptimizationApplication/python_scripts/topology_optimizer_factory.py

```diff
@@ -0,0 +1,370 @@
+# ==============================================================================
+#  KratosTopologyOptimizationApplication
+#
+#  License:         BSD License
+#                   license: TopologyOptimizationApplication/license.txt
+#
+#  Main authors:    Philipp Hofer, https://github.com/PhiHo-eng
+#                   Erich Wehrle, https://github.com/e-dub
+#  based on original file from
+#                   Baumgärtner Daniel, https://github.com/dbaumgaertner
+#                   Octaviano Malfavón Farías
+#                   Eric Gonzales
+#
+# ==============================================================================
+
+# ------------------------------------------------------------------------------
+# Imports
+# ------------------------------------------------------------------------------
+
+# Making KratosMultiphysics backward compatible with python 2.6 and 2.7
+from __future__ import print_function, absolute_import, division
```

Comment:
```suggestion
```

GENERAL COMMENT: we dropped support for python2 over a year ago, hence all those can be removed